code/items/ammo/SMGAmmo.cs

```diff
@@ -0,0 +1,13 @@
+using Sandbox;
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_ammo_smg")]
+    partial class SMGAmmo : TTTAmmo
+    {
+        public override string Type => "smg";
```

Comment:
Do we really need Type? Can't we use `SMGAmmo` or "SMGAmmo.Name" (which is set to the name of the lib) to avoid redundancy and string issues?